bin/utils/release_version_update_docs.sh

```diff
@@ -41,7 +41,6 @@ declare -a files=("modules/openapi-generator-maven-plugin/README.md"
                   "modules/openapi-generator-gradle-plugin/README.adoc"
                   "modules/openapi-generator-gradle-plugin/gradle.properties"
                   "modules/openapi-generator-gradle-plugin/samples/local-spec/gradle.properties"
-                  "modules/openapi-generator-gradle-plugin/samples/local-spec/build.gradle"
```

Comment:
@OpenAPITools/generator-core-team FYI. I've removed this from the update script as there's nothing to update in that file.